view/frontend/templates/js/replacejs.phtml

```diff
@@ -202,6 +202,13 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
     var expectCartRendering = true;
     var waitingForResolvingPromises = false;
 
+    var BoltState = {
```

Comment:
I am going to move other vars into BoltState in following PRs to have one global PR for interactions between plugin and extensions

Reply:
Do you mean one global variable? This seems like a reasonable extraction, but I think it deserves a block comment explaining the purpose of the var so others know when to add things to it.